test-fixture-mocha.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   test-fixture-mocha.js
+ */
```

Comment:
This seems like it is missing type information about the global `fixture` helper.

Reply:
I declared a new function in global context, as it was not possible to add this to a specific Mocha definition. Nonetheless, this is typed now.